packages/melos/lib/src/commands/run.dart

```diff
@@ -20,6 +20,36 @@ mixin _RunMixin on _Melos {
       );
     }
 
+    if (script.steps != null && script.steps!.isNotEmpty) {
+      //TODO(jessica): Reevaluate the design decision regarding execution
+      // options (e.g., concurrency) for scripts with steps.
+      // Current concern: Allowing execution options for scripts with steps
+      // might introduce complexity and unpredictability, especially if nested
+      // scripts also utilize concurrency.
```

Comment:
This TODO can be removed, I don't think we'd ever want to enable concurrency for groups, it'll be way too messy. :sweat_smile: